modules/ROOT/pages/type-definitions/directives/index.adoc

```diff
@@ -1,11 +1,182 @@
 [[directives]]
 :description: This page lists all directives available in the Neo4j GraphQL Library.
+
 = Directives
 :page-aliases: directives.adoc
 
 The Neo4j GraphQL Library provides the following directives to be used whilst defining types:
 
-[cols="1,3"]
+== Database mapping
+
+.Database mapping directives
+[cols="2,5"]
+|===
+| Directive | Description
+
+| xref::/type-definitions/directives/database-mapping.adoc#_relationship[`@relationship`]
+| Configure  xref::/type-definitions/types/relationships.adoc[relationships] between object types.
+
+| xref::/type-definitions/directives/database-mapping.adoc#_relationship_properties[`@relationshipProperties`]
+a| Required to help you distinguish between interfaces which are used for relationship properties, and otherwise.
+Can only be used on interfaces, as per its definition:
```

Comment:
Do you think it's important to have this quote here in the index of the directives section? No other item has it, and maybe it could be mentioned in the page for `@relationshipProperties` itself instead.